src/Controls/src/Core/Compatibility/Android/PlatformSizeService.cs

```diff
@@ -8,7 +8,13 @@ class PlatformSizeService : IPlatformSizeService
 	{
 		public SizeRequest GetPlatformSize(VisualElement view, double widthConstraint, double heightConstraint)
 		{
-			return Platform.GetNativeSize(view, widthConstraint, heightConstraint);
+			if (widthConstraint > 0 && heightConstraint > 0)
+			{
+				return view.Handler?.GetDesiredSize(widthConstraint, heightConstraint) ??
+					new SizeRequest();
+			}
+
+			return new SizeRequest();
```

Comment:
The same as above. If renderers are all secretly handlers, then we can just delete these services because they just exist for renderer logic.

Reply:
Same comment as above :-)